src/scorecard.ts

```diff
@@ -17,8 +17,16 @@ export async function getScorecardLevels(
       repositoryUrl = repositoryUrl.replace('https://', '')
     }
 
+    // Handle the special case for GitHub Actions, where the repository URL is null
+    if (ecosystem === 'actions') {
+      // The package name for GitHub Actions in the API is in the format `owner/repo/`, so we can use that to get the repository URL
+      // If the package name has more than 2 slashes, it's referencing a sub-action, and we need to strip the last part out
+      const parts = packageName.split('/')
+      repositoryUrl = `github.com/${parts[0]}/${parts[1]}` // e.g. github.com/actions/checkout
+    }
+
```

Comment:
🤔 nit: would be good to explicitly check this output in the new test case?

Reply:
Hmm, the repositoryUrl isn't exposed outside of the function. Thoughts on how to bootstrap that into the tests? Was trying to avoid side effects on the Change object by using a local variable.

Reply:
Sorry late getting back to this. May the easiest play is to factor out 21-26 into a function that accepts a `Change` and outputs a `repositoryUrl` (or not) that line 29 can check, then test that in isolation with some Actions-flavored `Change`s?

Again not the end of the world, since it's already landed 👍